modules/tide_publication/src/Controller/PublicationResource.php

```diff
@@ -181,53 +182,110 @@ public function getHierarchy(EntityInterface $entity, Request $request) {
    *   The hierarchy.
    */
   protected function buildHierarchy(ContentEntityInterface $entity, NestedSetStorage $storage, CacheableMetadata $cache, $weight, array &$flatten_hierarchy, $site = NULL) {
-    $resource_type = $this->resourceTypeRepository->get($entity->getEntityTypeId(), $entity->bundle());
+    // Try cache first.
+    $cid = "tide_publication:hierarchy:{$entity->uuid()}:{$weight}:" . ($site ? $site->id() : 'null');
```

Comment:
Why do these get surrounded by `{}`?

Reply:
Hi @anthony-malkoun , it just clearly defines variable boundary in this string.